modules/cas/lib/Auth/Source/CAS.php

```diff
@@ -78,6 +78,29 @@ public function __construct($info, $config) {
 		}else{
 			throw new Exception("cas login URL not specified");
 		}
+
+
+		// Check for the required ldap configuration 
+		if(!isset($this->_ldapConfig['servers'])) {
```

Comment:
This makes sense because I see that above a "ldap" section in the config is already enforced. However, I noticed that in finalStep(), `_ldapconfig['servers']` is checked whether it is set, making it essentially optional. I think we should either make `_ldapconfig['ldap']['servers'] required, or not (and specify which it is in the docs) but the current code is ambivalent.